homeassistant/components/tractive/entity.py

```diff
@@ -18,7 +18,7 @@ def __init__(
         self._attr_device_info = DeviceInfo(
             configuration_url="https://my.tractive.com/",
             identifiers={(DOMAIN, tracker_details["_id"])},
-            name=f"Tractive ({tracker_details['_id']})",
+            name=f"{trackable['details']['name']} ({tracker_details['_id']})",
```

Comment:
I changed the device name from `Tractive (<tracker ID>)` to `<pet name> (<tracker ID>)`, so the entity names would be something like `Karmel (ABCDEFS) Daily goal`. I'm not sure if the tracker ID is needed here.

Reply:
Honestly, I don't think people would use/be aware/care about their tracker IDs... They care about tracking their pets (in which case the name would be sufficient).